dashboard/db/schema.rb

```diff
@@ -301,7 +301,7 @@
     t.index ["follower_id"], name: "index_code_review_group_members_on_follower_id"
   end
 
-  create_table "code_review_groups", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
+  create_table "code_review_groups", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
Are you sure we want the other changes in to this file? I'd expect to see just the ones related to the migration on line 688.

Reply:
Thanks for catching that! Some discrepancies between my local database configuration and our managed environments’ databases caused this. I’ve reverted those errant schema changes.